### PR TITLE
Create an ADR for policy server

### PR DESCRIPTION
**Pull request by XanderStrike**
[#173270851](https://www.pivotaltracker.com/story/show/173270851)

This PR adds an ADR to describe our approach for handling Network Policy for App to App communication.

### CODE REVIEW

File: doc/architecture-decisions/0018-create-policy-server.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+Date: 2020-10-27
+
+## Status
+
+Accepted
+
+## Context
+
+Well implemented 12 factor apps typically include backend services who never
```
Comment (dolfolife) [inline doc/architecture-decisions/0018-create-policy-server.md]: should we include the link for https://12factor.net/ ?

Comment (XanderStrike) [reply doc/architecture-decisions/0018-create-policy-server.md]: I wasn't sure I wanted to reference 12 factor at all actually but I can't think of a more general way to phrase it. Maybe "service oriented apps" or something?

Comment (dolfolife) [reply doc/architecture-decisions/0018-create-policy-server.md]: that make sense yes or cloud native :) I just found 12 factor app being very attached to that manifest :)

Comment (XanderStrike) [reply doc/architecture-decisions/0018-create-policy-server.md]: Boom done thanks

**Comment by rosenhouse** [comment]
Maybe call out explicitly that we're not providing a migration path from the old network-policy server database.

**Pull request merged**